plugin.video.spurs-tv/resources/language/English/strings.xml

```diff
@@ -9,7 +9,9 @@
   <string id="30014">All</string>
```

Comment:
Next update please use "strings.po" in plaxce of of "strings.xml", which is deprecated.